app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -230,21 +230,25 @@ def process_enrollment_response(enrollment, response)
 
   def send_verified_email(user, enrollment)
     user.confirmed_email_addresses.each do |email_address|
+      # rubocop:disable IdentityIdp/MailLaterLinter
       UserMailer.in_person_verified(
         user,
         email_address,
         enrollment: enrollment,
-      ).deliver_now_or_later(**mail_delivery_params)
+      ).deliver_later(**mail_delivery_params)
```

Comment:
should we update the lint to detect if we send params to `deliver_now_or_later` because those will get ignored?

Reply:
You mean if we pass params to `deliver_now_or_later` adding a warning that says something like "Parameters sent to deliver_now_or_later will be ignored if the deliver_mail_async flag is off"?

Reply:
yup! (either that, or change the behavior inside `deliver_now_or_later` to use the params if they are passed)